tests/js/spec/components/assigneeSelector.spec.jsx

```diff
@@ -214,16 +207,13 @@ describe('AssigneeSelector', () => {
 
     userEvent.click(screen.getByText(`${USER_1.name} (You)`));
 
-    expect(await screen.findByTestId('loading-indicator')).toBeInTheDocument();
```

Comment:
fixes by removing it 😂 

Did this behavior change?

Reply:
the loading indicator only appears momentarily, if the tests are running quickly it doesn't seem to always find it